cloud/pkg/cloudstream/config/config_test.go

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2020 The KubeEdge Authors.
```

Comment:
The copyright date should be updated to 2022.

Reply:
updated